Lib/venv/__init__.py

```diff
@@ -158,6 +158,7 @@ def create_configuration(self, context):
                 incl = 'false'
             f.write('include-system-site-packages = %s\n' % incl)
             f.write('version = %d.%d.%d\n' % sys.version_info[:3])
+            f.write(f'prompt = {context.prompt}\n')
```

Comment:
Dealing with leading/trailing whitespace could get interesting here. All the other values can be trimmed at both ends, while this one (currently) must have one space removed from the start.

Perhaps we should insert the `repr` of the prompt instead? Or at least double-quote it and escape internal quotes and backslashes?

Reply:
Thanks for the suggestion.